tests/tiup/upgrade-from-v2.sh

```diff
@@ -27,10 +27,15 @@ function deploy_previous_v2() {
 function migrate_in_previous_v2() {
 	exec_full_stage
 
-	# v2.0.0 doesn't implement relay log, and enable-gtid for MariaDB has a bug
-	if [[ "$PRE_VER" == "v2.0.0" ]]; then
-		sed -i "s/enable-relay: true/enable-relay: false/g" $CUR/conf/source2.yaml
-		sed -i "s/enable-gtid: true/enable-gtid: false/g" $CUR/conf/source2.yaml
+	# v2.0.0 doesn't implement relay log.
+	# v2.0.1 downgrading has a problem, we use config import which create sources and then create tasks, so for a cluster
```

Comment:
maybe we can let `config export` get location of `getMinLocInAllSubTasks` and write it to source config's relay location? @GMHDBJD